src/main/java/com/splunk/ibm/mq/metricscollector/ChannelMetricsCollector.java

```diff
@@ -106,7 +106,10 @@ public void publishMetrics() {
                       metricCreator.createMetric(
                           metrickey, metricVal, wmqOverride, channelName, metrickey);
                   responseMetrics.add(metric);
-                  if ("Status".equals(metrickey) && metricVal == 3) {
+                  if ("Status".equals(metrickey)
+                      && metricVal != CMQCFC.MQCHS_RETRYING
+                      && metricVal != CMQCFC.MQCHS_STOPPED
+                      && metricVal != CMQCFC.MQCHS_STARTING) {
```

Comment:
this is a semantic change. intentional?

Reply:
yes it is the change. Here is the doc https://www.ibm.com/docs/en/ibm-mq/9.2.x?topic=states-current-active

![image](https://github.com/user-attachments/assets/43b5cb13-2214-4df9-b9c4-f49240091f49)

Reply:
@atoulme Thanks for the diagram.  I personally wouldn't mind putting a comment with the ibm link 😅   that's just me